keras_rcnn/heads/resnet.py

```diff
@@ -0,0 +1,96 @@
+import keras
+
+
+def ResHead(classes, mask=False):
+    """Resnet heads as in Mask R-CNN."""
+    def f(x):
+        if keras.backend.image_data_format() == "channels_last":
+            channel_axis = 3
+        else:
+            channel_axis = 1
+
+        y = keras.layers.TimeDistributed(
```

Comment:
@0x00b1 That would be great! I am a bit busy these days. I will get back to this asap.